adapters/smaato/smaato.go

```diff
@@ -545,20 +533,6 @@ func makeImpExt(impExtRaw *json.RawMessage) (json.RawMessage, error) {
 	}
 }
 
-func setBannerDimension(banner *openrtb2.Banner) (*openrtb2.Banner, error) {
```

Comment:
@Enigo  any particular reason to remove `setBannerDimension` >

Reply:
Bcz it's not needed, server-side we can process w/h from both banner and banner.format, so there's no need to duplicate them.

Reply:
@onkarvhanumante 
can this PR be merged with just one approve from you? 

@Sonali-More-Xandr seems to be unavailable